h5p-bildetema/src/components/Menu/Menu.tsx

```diff
@@ -50,7 +50,7 @@ export const MenuItem = forwardRef<HTMLElement, MenuItemProps>(
   ({ label, icon, onClick, ...props }, ref) => (
     <HMenuItem {...props} ref={ref}>
       <Button onClick={onClick} className={styles.menuItemButton}>
-        {icon}
+        <span aria-hidden="true">{icon}</span>
```

Comment:
What is the purpose of the aria-hidden function? Shouldn't the icon always be there?

Reply:
If there is an icon, screen readers will announce "image", but we only want the text inside the `button` to be announced, not "image Copy link" for instance when there is an icon placed before the button text. When there are icons or other images that are only decorative they should have `aria-hidden` set to true.